src/App.js

```diff
@@ -0,0 +1,65 @@
+import React,{useState } from "react";
+import { Button } from './components/Button/Button';
+
+export default function App() {
+  const [users, setUsers] = useState();
+  const [error, setError] = useState();
+
+  const handleClick = async () => {
+   
+    try {
+      const response = await fetch(
+        "https://jsonplaceholder.typicode.com/users"
+      );
+      if (response.ok) {
+        const data = await response.json();
+        
+        getAllAvatar(data);
+      } else {
+          setError("there was an error getting the user");
+      }
+    } catch (error) {
+        setError("We couldn't make the request to get the user");
+    }
+  }
+
+    const getAllAvatar = async (data) => {
+      let newUsersList = [];
+      try {
+        for (let i = 0; i < data.length; i++) {
+          const response = await fetch(
+            "https://avatars.dicebear.com/api/male/john.svg?background=%230000ff"
```

Comment:
User the user here instead of the john hardcoded

Reply:
> User the user here instead of the john hardcoded

She had to use it like this because the proper endpoint was throwing an error. So I told her to leave it like this till the API is working again.